### PR TITLE
Docs for order-utils and test-utils

### PR DESCRIPTION
**Pull request by alicevhenshaw**

### CODE REVIEW

File: helper-libraries/orders-and-quotes.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    'r' in order['signature'] &&
+    's' in order['signature'] &&
+    'v' in order['signature']
+  )
+}
+```
+
+## `checkOrder`
+
+Checks whether the swap contract would accept or reject a specified order. THis includes checking wallet balances, approvals, authorizations, signatures, expiry and nonces,
```
Comment (ejwessel) [inline helper-libraries/orders-and-quotes.md]: `THis` -> `This`

File: helper-libraries/padding-locators.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## `padAddressToLocator`
+
+Pads a given Ethereum address into a bytes32 to be used as a locator on the Indexer. Ethereum addresses are 190 bits and a bytes32 is 256 bits.
+
+| Param      | Type       | Description                                 |
+| :--------- | :--------- | :------------------------------------------ |
+| `address`  | `address`  | The address to be padded to a bytes32.      | 
+
+**Returns** the address padded with a tail of 66 0s.
```
Comment (ejwessel) [inline helper-libraries/padding-locators.md]: `the address padded with a tail of 66 0s.` -> `the address right padded with 66 0s.>

File: helper-libraries/testing-token-balances.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| :----------- | :-------------------- | :---------------------------------------------- |
+| `account`    | `address`             | The address of the token owner.                 | 
+| `withdrawer` | `address`             | The address of the token spender.               | 
+| `balances`   | `[[address, integer]]`| Pairs of token address with expected allowance. |
+
+e.g. to verify that the Swap contract has allowance to transfer 500 AST and 20 WETH owned by Alice:
+
+`await allowances(aliceAddress, swapAddress, [[ASTaddress, 500], [WETHaddress, 20]])`
+
+**Returns** `true` if all allowances are correct.
```
Comment (poiresel) [inline helper-libraries/testing-token-balances.md]: if all allowances are satisfied.

File: helper-libraries/hashing-orders.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+When an order is submitted to the Swap contract, a signature of the order is provided with it. This signature is constructed by calculating the hash of the order, before signing this hash. Due to the use of sub-objects in an order, there are many different ways of hashing an order - however the smart contract only accepts one. This helper library provides functions to calculate hashes used in various parts of the Airswap Protocol.
```
Comment (poiresel) [inline helper-libraries/hashing-orders.md]: Cryptographically signing an order gives security to the signer that the sender of the order cannot modify the order contents.

File: helper-libraries/hashing-orders.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+When an order is submitted to the Swap contract, a signature of the order is provided with it, which gives security to the signer that the sender of the order cannot modify the order's contents.. This signature is constructed by calculating the hash of the order, before signing this hash. Due to the use of sub-objects in an order, there are many different ways of hashing an order - however the smart contract only accepts one. This helper library provides functions to calculate hashes used in various parts of the Airswap Protocol.
```
Comment (ejwessel) [inline helper-libraries/hashing-orders.md]: ```odify the order's contents..``` double period.

Comment (dmosites) [reply helper-libraries/hashing-orders.md]: Capitalize "AirSwap", though would simply say "...hashes used throughout the protocol."

**Pull request merged**